src/cargo/core/package.rs

```diff
@@ -39,9 +39,7 @@ pub const MANIFEST_PREAMBLE: &str = "\
 # with all versions of Cargo and also rewrite `path` dependencies
 # to registry (e.g., crates.io) dependencies
 #
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
+# If you are reading this file be aware that the upstream Cargo.toml
 # will likely look very different (and much more reasonable)
```

Comment:
Should this say something like “look for cargo.toml.orig” for the original Cargo.toml? Looks like the wording predates the PR where we starred to package original cargo,toml as well.